product_docs/docs/pgd/5.6/commit-scopes/commit-scopes.mdx

```diff
@@ -175,4 +176,40 @@ SELECT bdr.alter_node_group_option(
 );
 ```
 
+### ORIGIN_GROUP syntax sugar
```

Comment:
Do not refer to it as syntax sugar. From the users POV it is a useful function and they know nothing of the internals of the syntax that would make it sugary.

Reply:
Oh I see good point.